File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    let split = rsp.split("");

    for (let i = 0 ; i< rsp.length; i++) {
        if (split[i] === "2") split[i] = "0";
        else if (split[i] === "5") split[i] = "2";
        else split[i] =  "5"
    }
    return split.join("");
}

File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.cs
using System;

public class Solution {
    public string solution(string rsp)
            {
                string answer = "";

                for (int i = 0; i < rsp.Length; i++)
                {
                    if (rsp[i].ToString().Equals("2"))
                        answer += "0";
                    else if (rsp[i].ToString().Equals("0"))
                        answer += "5";
                    else
                        answer += "2";
                }

                return answer;
            }
}